scripts/runtime-audit.ps1

```
@@ -0,0 +1,227 @@
+    }
+    return @($output)
+}
+
+function Convert-RgJsonLine {
+    param([Parameter(Mandatory = $true)][string]$Line)
+
+    if ([string]::IsNullOrWhiteSpace($Line)) {
+        return $null
+    }
+
+    $event = $Line | ConvertFrom-Json
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'event' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)